packages/utils/tests/enumUtils.test.ts

```diff
@@ -0,0 +1,36 @@
+import { getKeyByValue, getValueByKey } from '../src/enumUtils';
+
+enum NumberEnum {
+    ZERO,
+    ONE,
+}
+
+enum StringEnum {
+    ZERO = '0',
+    ONE = '1',
+}
+
+describe('enumUtils', () => {
+    it('getKeyByValue', () => {
+        expect(getKeyByValue(NumberEnum, 0)).toEqual('ZERO');
+        expect(getKeyByValue(NumberEnum, 1)).toBeUndefined();
+        // @ts-expect-error string value should not be accepted
+        expect(getKeyByValue(NumberEnum, '1')).toBeUndefined();
+
+        expect(getKeyByValue(StringEnum, StringEnum.ZERO)).toEqual('ZERO');
+        // @ts-expect-error number value should not be accepted
+        expect(getKeyByValue(StringEnum, 1)).toBeUndefined();
+    });
+
+    it('getValueByKey', () => {
+        expect(getValueByKey(NumberEnum, 'ONE')).toEqual(1);
+        // @ts-expect-error number value should not be accepted
+        expect(getValueByKey(NumberEnum, 1)).toBeUndefined();
+        // @ts-expect-error not a key
+        expect(getValueByKey(NumberEnum, '1')).toBeUndefined();
+
+        expect(getValueByKey(StringEnum, 'ONE')).toEqual('ONE');
```

Comment:
I don't get it, why it equals 'ONE' and not '1'?

Reply:
its a `StringEnum` where keys and values are strings

Reply:
very good catch 🥇 im glad that someone acutaly read this :) but why CI didnt catch that? it obviously should fail on validation (it is/was failing locally)

Reply:
Because CI failed on type check so it didn't proceed unit tests https://github.com/trezor/trezor-suite/actions/runs/3480708189/jobs/5820853484#step:13:343